prow/cmd/pipeline/main.go

```diff
@@ -64,18 +64,19 @@ func (o *options) parse(flags *flag.FlagSet, args []string) error {
 	flags.BoolVar(&o.allContexts, "all-contexts", false, "Monitor all cluster contexts, not just default")
 	flags.StringVar(&o.totURL, "tot-url", "", "Tot URL")
 	flags.StringVar(&o.kubeconfig, "kubeconfig", "", "Path to kubeconfig. Only required if out of cluster")
-	flags.StringVar(&o.config, "config", "", "Path to prow config.yaml")
+	flags.StringVar(&o.configPath, "config", "", "Path to prow config.yaml")
```

Comment:
We should add a validation that this is set

Reply:
I didn’t add validation here because I call config.ConfigPath later that sets the default config path if it was not set on the command line.

It is what is done in other components (hook, deck, etc...) and I thought it would be more consistent to do the same thing here. 

I can add validation if you prefer though.

Reply:
It is bad that other components default this field. We only continue to do that for backwards compatibility and it looks like that should have been deprecated by now so please don't add more components that use it: https://github.com/kubernetes/test-infra/blob/7ac63bdbab0f3666497bede973d7703bb30e9cd4/prow/config/config.go#L1324
Validation is preferable.

Reply:
Sorry, didn't know that... I should probably follow more closely sig meetings where we communicate this -_-"

Reply:
Thank you for your explanation, i removed the call to `config.ConfigPath` and added validation.